src/UI/Radio.elm

```diff
@@ -89,6 +95,7 @@ type alias Options id =
     { selected : Maybe id
```

Comment:
no change needed here, but radios mostly used with a default value already selected, and since you can't deselect it we could improve the API by getting rid of the maybe here

Reply:
That's true, but we would have to change the whole API in fact, because then we should not be able to create a group with an empty options list